data/remnant/remnant ships.txt

```diff
@@ -230,6 +230,324 @@ ship "Albatross" "Albatross (Ember Shield)"
 	turret
 
 
+
+ship "Modified Dromedary"
+	sprite "ship/modified dromedary"
+	thumbnail "thumbnail/modified dromedary"
+	attributes
+		licenses
+			Remnant Capital
+		category "Transport"
+		"cost" 57600000
+		"shields" 15100
+		"hull" 9600
+		"required crew" 95
+		"bunks" 300
+		"mass" 2140
+		"drag" 15.1
+		"heat dissipation" .75
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"weapon capacity" 270
+		"engine capacity" 195
```

Comment:
For background, this ship's story is that it belonged to a captain who owned & flew it for several decades. A captain who was well-versed in the engineering and mechanical trades and had a fondness for modifying their ship, and took advantage of the fact that it can, in fact, carry sufficient heavy industrial equipment to manufacture fairly major structural components. Said captain did a lot of exploratory work, often serving as a mobile station for longer-term research projects that need a platform to remain more or less in-place for several weeks or even months.

If density-per-hardpoint is the same, but total hardpoints is less, then it's total combat capability is lower. This isn't intended to be an actual weaker ship than the Bactrian, it's more of an alternate configuration that prioritizes stuff that isn't quite so combat focused. But yes, as you point out, it is quite capable of being fairly close to it if one puts the effort in. It'll never match the Bactrian's DPS potential, nor its absolute "take simultaneous hits" capability. But its ability to run and outlast opponents is at least equal and possibly superior, depending on outfitting.

It's significantly higher mass, for instance, means that it will be significantly less vulnerable to being "pinned down" and blocked from jumping to hyperspace. It does take a lot more engine to move, though.